lib/complete/complete.go

```diff
@@ -133,6 +133,9 @@ reset:
 		return c.suggestLeafCommandOptions(), nil
 	case !arg.Completed:
 		// Currently typing a sub command.
+		if arg.HasFlag {
+			return c.suggestLeafCommandOptions(), nil
```

Comment:
Wouldn't this prioritize showing flags instead of subcommands now? Or does it show both?

Reply:
@UmanShahzad It checks for the `-` at the start. If it's a flag, only then does it get prioritized.

Here is the current behavior.
If users type without `-` or just press the tab, the following options will show up.

![image](https://user-images.githubusercontent.com/109338796/230639108-8e9f7962-3e2c-4db7-bfd9-7167773fcfaf.png)

if the user starts with `-`, the following options will show up.

![image](https://user-images.githubusercontent.com/109338796/230639339-26023296-eff2-4d7e-9eea-6e4ea05a3176.png)